src/layout/actions/layout.rs

```diff
@@ -224,6 +254,36 @@ impl LayoutTree {
         self.validate();
     }
 
+    /// Gets the active container and toggles it based on the following rules:
+    /// * If horizontal, make it vertical
+    /// * else, make it horizontal
+    /// This method does *NOT* update the actual views geometry, that needs to be
+    /// done separately by the caller
+    pub fn toggle_active_horizontal(&mut self) {
```

Comment:
good point. Also I should look into why the view geometry isn't actually updatedd by this method...